jobs/v3/snippets/basic_company_sample.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"golang.org/x/oauth2/google"
+	jobs "google.golang.org/api/jobs/v3"
+)
+
+const requestDeadline = 10 * time.Second	
+
+func prettyFormat(v interface{}) string {
```

Comment:
Normally, for samples we use the default formatting to print (`%v` or `%q`). Is there a reason this needs to be JSON? I'm concerned about users needing to understand something unrelated.